package.json

```
@@ -5,8 +5,8 @@
   "main": "src/index.js",
   "type": "module",
   "scripts": {
-    "test": "c8 --exclude 'test/fixtures/**' mocha -i -g Integration",
-    "integration-ci": "c8 --exclude 'test/fixtures/**' mocha -g Integration",
+    "test": "c8 --exclude 'test/fixtures/**' --exclude 'src/template/fastly-runtime.js' mocha -i -g Integration",
+    "integration-ci": "c8 --exclude 'test/fixtures/**' --exclude 'src/template/fastly-runtime.js' mocha -g Integration",
```

```suggestion
    "test": "c8 --exclude 'test/fixtures/**' mocha -i -g Integration",
    "integration-ci": "c8 --exclude 'test/fixtures/**' mocha -g Integration",
```

do not exclude the template code from test coverage measurement